synapse/tests/test_model_infotech.py

```diff
@@ -1545,6 +1573,95 @@ async def test_infotech_cpes(self):
             self.len(1, await core.nodes('it:sec:cpe:v2_2=cpe:/a:vertex:synapse'))
             self.len(1, await core.nodes('it:sec:cpe:v2_2=cpe:2.3:a:vertex:synapse:*:*:*:*:*:*:*:*'))
 
+            # Test cpe22 -> cpe23 escaping logic
+            norm, info = cpe23.norm('cpe:/a:%21')
+            self.eq(norm, 'cpe:2.3:a:\\!:*:*:*:*:*:*:*:*:*')
+
+            norm, info = cpe23.norm('cpe:/a:%5c%21')
+            self.eq(norm, 'cpe:2.3:a:\\!:*:*:*:*:*:*:*:*:*')
+
+            norm, info = cpe23.norm('cpe:/a:%5cb')
+            self.eq(norm, 'cpe:2.3:a:\\\\b:*:*:*:*:*:*:*:*:*')
+
+            norm, info = cpe23.norm('cpe:/a:b%5c')
+            self.eq(norm, 'cpe:2.3:a:b\\\\:*:*:*:*:*:*:*:*:*')
+
+            norm, info = cpe23.norm('cpe:/a:b%5c%5c')
+            self.eq(norm, 'cpe:2.3:a:b\\\\:*:*:*:*:*:*:*:*:*')
+
+            norm, info = cpe23.norm('cpe:/a:b%5c%5cb')
+            self.eq(norm, 'cpe:2.3:a:b\\\\b:*:*:*:*:*:*:*:*:*')
+
+            # Examples based on customer reports
```

Comment:
These two sets of nodes should be the same idens/buids, correct?

Reply:
The dups should be the same, yeah.